terraso_backend/apps/core/management/commands/loadbackup.py

```diff
@@ -39,7 +39,8 @@ def _find_latest_backup_dir(directory):
         if len(files) < 2:
             raise RuntimeError(f"Couldn't find any backup files in {directory}")
         # default is to sort ascending, so latest files at end
-        return files[-2:]
+        data_file, migrations_file = files[-2:]
+        return data_file, migrations_file
```

Comment:
question: what else is in the files array?

Reply:
It's possible that we have more than one set of backups in the bucket, for convenience or if there is a backup we want to save for whatever reason. Do you think there is a better way to handle this?